code/_onclick/click_alt.dm

```diff
@@ -6,6 +6,11 @@
 /mob/proc/base_click_alt(atom/target)
 	SHOULD_NOT_OVERRIDE(TRUE)
 
+	if(HAS_TRAIT(src, TRAIT_MECH_PILOT) && istype(loc, /obj/vehicle/sealed/mecha))
+		if(target == loc)
+			target.click_alt(src)
+		return
```

Comment:
Why have a snowflake check? Can we not handle this via signals?

Reply:
`COMSIG_MOB_ALTCLICKON`? (Which actually seems to not be sent anywhere anymore despite still being defined and registered)